src/descriptor.rs

```diff
@@ -148,10 +152,145 @@ impl Descriptor {
     }
 
     /// Constructs a new [`Descriptor`] from a byte [`Vec`].
+    ///
+    /// # Warning
```

Comment:
I don't think this really warrants a warning tag, just a note that callers who don't want to muck about with tags and byte serialization can use the convenience constructors instead.

Reply:
Will convert to `# Note`.

Reply:
Done in df5ff0782e13c20029cdc36b7942ad797a80d9d9

gave up on H1